client/sites-dashboard/components/no-sites-message.tsx

```diff
@@ -8,15 +8,32 @@ const NoSitesLayout = styled( EmptyContent )`
 	flex-direction: column;
 	justify-content: center;
 	align-items: center;
+
+	${ ( { illustration } ) =>
+		! illustration && {
+			marginTop: '10%',
+		} }
+
+	.empty-content__line {
+		margin-bottom: 20px;
+	}
+
+	.empty-content__illustration {
+		margin-bottom: 30px;
+	}
 `;
 
 const SecondaryText = styled.p`
-	max-width: 650px;
+	max-width: 550px;
+	font-size: 14px;
 `;
 
 const Title = styled.div`
-	margin-top: 50%;
+	font-family: 'Recoleta';
```

Comment:
We use the `onboarding-font-recoleta` mixin to include Recoleta font and fallbacks in SASS files.  Should we handle it somehow in styled components too?

Reply:
I would lean toward using the mixin if possible to keep that single source of truth.

Reply:
@vykes-mac I've checked it, and we can't use SASS mixins in the styled components.

Should we declare the font along with all fallbacks to match the mixin?

```'Recoleta', 'Noto Serif', Georgia, 'Times New Roman', Times, serif```

Also, do we want to introduce mixin-like behavior so that we would extract commonly used CSS variables to the separate JS file?

@zaguiini @p-jackson, what would you recommend here?

Reply:
I can't seem to find where that set of fallback fonts is used in a mixin ...

Using CSS variables is an interesting idea, since we'd be able to reference those variable names from within Emotion. But I'm not sure whether we can rely on CSS variable support

I think will just be easiest to copy and paste the font face fallbacks for now.

It might be nicer to have a `.js` file that sites beside the `.scss` file that defines the mixin (wherever that is) and the `.js` could export a style object. That way Emotion could consume that style object. Hopefully if one of the files got updated, the other file would get updated at the same time. But yeah, I already see the `'Recoleta'` string is copy and pasted at a number of places in the codebase already.

Reply:
We do it in https://github.com/Automattic/wp-calypso/blob/5f0de103b3aff2fdf9397686a0846c8f80600cf0/packages/typography/styles/variables.scss#L15 

I've changed it to use the full fallback font for now.

Reply:
Of course! I got confused and forgot that `$serif` itself was a variable 😅